src/hotspot/share/cds/filemap.cpp

```diff
@@ -850,7 +865,12 @@ bool FileMapInfo::validate_boot_class_paths() {
     if (relaxed_check) {
       return true;   // ok, relaxed check, runtime has extra boot append path entries
     } else {
-      mismatch = true;
+      ResourceMark rm;
```

Comment:
Is the ResourceMark necessary here?

Reply:
Yes, it is needed for `ClasspathStream::get_next` where it does `NEW_RESOURCE_ARRAY`.